templates/registration/profile.html

```diff
@@ -12,8 +17,13 @@ <h2 class="profile-header">Good morning {% firstof user.get_full_name user.get_u
             <h2>Portfolio</h2>
             <p>Here are the artists that you have invested in:</p>
             <ul>
-                {% for artist in artists %}
-                    <li><a href="{% url 'artist' slug=artist.slug %}">{{ artist.name }}</a></li>
+                <div class="row small-up-1 medium-up-2 large-up-4 artistlist">
```

Comment:
This section looks like it got messed up. It would be easier to see what's wrong if the indenting was correct.
From a quick glance though, this `<div>` looks like it's not being closed.